net-proxy/trojan/trojan-1.8.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="An unidentifiable mechanism that helps you bypass GFW"
+HOMEPAGE="https://github.com/trojan-gfw/trojan"
+SRC_URI="https://github.com/trojan-gfw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="mysql test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=dev-libs/boost-1.54.0:=
+	>=dev-libs/openssl-1.0.2:=
+	mysql? ( dev-db/mysql-connector-c:= )
+	test? ( net-misc/curl >=dev-lang/python-3.4.5 )
```

Comment:
All uses of Python must go through one of python-* eclasses, as documented on https://wiki.gentoo.org/wiki/Project:Python/Eclasses